environments.json

```diff
@@ -1458,10 +1458,25 @@
   "node7": {
     "full": "Node.js",
     "family": "Node.js",
-    "short": "Node 7",
+    "short": "Node 7.0-7.5",
```

Comment:
The changes caused by the v8 version change are already covered by the `"equals": "chrome55"` field. All non-v8 specific things seem to be equal.